modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -98,6 +108,19 @@ def upload_documents
 
       private
 
+      def generate_multi_image_pdf(image_list)
```

Comment:
My initial thought with PDF generation was that it would also be in a background job. But if it's not too intensive it makes sense for generation to succeed then return a 200 to the user and upload in the background (or fail and let the user know). 

What is collective size limit for images? Have you tried generating a PDF at that limit and if so any metrics on how long it takes?

Reply:
I'll try that, the limit is 50 MB I believe

Reply:
Could be good to have some statsd metrics to see how long things take on staging/prod.

Reply:
I tested a 50MB image which would take 6-8 seconds to finish. I tried smaller individual images (2 25MBs, 5 10MBs, etc), but the smaller each individual image was the faster the request finished. Also added statsd to record the time it takes so we can check it in upper envs.

Reply:
I think because of the upper bounds and phone cameras outputting larger and larger images that it could make send to move the PDF processing to a background job as well. If the images are valid and are saved successfully we could return a response to the user. This can be another ticket as staging testing will give us more data about processing times in a more production like env.